Requirements.md

```diff
@@ -0,0 +1,63 @@
+# Software Requirements Specifications for Spring 2020 "Intro to Software Engineering" *Turn* Project at Wright State University
+
+## Functional Requirements
+
+### [User Story A](Features.md "User Stories")
+
+Req. 1.0 (Jonathan): The game shall include a store that allows users to purchase items
+
+Req. 1.0.1 (Jonathan): The game shall include a store that allows user to purchase in-game items with in-game coins
+
+Req. 1.0.2 (Jonathan): The game shall include a store that allows a user to purchase 1-9 of a single item per transaction
+
+Req. 1.0.3 (Jonathan): The game shall not allow a user to purchase an item if they do not have the necessary amount of coins
+
+Req. 1.1 (Jonathan): The game shall include a store that allows users to sell items
+
+Req. 1.1.1 (Jonathan): The game shall include a store that allows users to sell in-game items for in-game coins
+
+Req. 1.1.2 (Jonathan): The game shall include a store that allows users to sell 1-9 of a single item per transaction
+
+Req. 1.1.3 (Jonathan): The game shall not allow a user to sell an item if they do not have the necessary amount of item(s) to sell
+
+### [User Story B](Features.md "User Stories")
+
+Req: 2.0 (Hayden): The game shall implement intellgent AI enemies
+
+Req: 2.0.1 (Hayden): The enemy AI shall not heal when at full health
+
+Req: 2.0.2 (Hayden): The enemy AI shall heal more frequently in battle when below 50% health
+ 
+### [User Story C](Features.md "User Stories")
+
+Req. 3.0 (Hayden): The game shall include multiple player classes
+
+Req. 3.0.1 (Hayden): The game shall include a special attack in battle for the respective player classes
+
+Req. 3.0.1.1 (Hayden): The special attack for the respective player class shall inflict damage relevant to their class.
+
+Req. 3.1 (Hayden): The game shall include more player classes than the default class
+
+### [User Story D](Features.md "User Stories")
+
+Req. 4.0 (Jonathan): The game shall implement an armor system
+
+Req. 4.0.1 (Jonathan): The game shall implement an armor system that reduces damage dealt to the Player
+
+Req. 4.1 (Jonathan): The game shall implement an X-Attack System
```

Comment:
Define X-Attack System in a brief introduction. Consider stating req. as "The X-Attack system shall..."